okhttp/src/main/java/okhttp3/Response.kt

```diff
@@ -353,7 +424,17 @@ class Response internal constructor(
           checkNotNull(request) { "request == null" },
           checkNotNull(protocol) { "protocol == null" },
           checkNotNull(message) { "message == null" },
-          this)
+          code,
+          handshake,
+          headers.build(),
+          body,
+          networkResponse,
+          cacheResponse,
+          priorResponse,
+          sentRequestAtMillis,
+          receivedResponseAtMillis,
```

Comment:
Should we check `sentRequestAtMillis >= 0` and `receivedResponseAtMillis >= 0` like we do `check(code > 0) { "code < 0: $code" }`?

Reply:
No, 0 isn’t special for timestamps